shared/chat/conversation/input-area/normal/index.tsx

```diff
@@ -372,9 +372,12 @@ const ConnectedPlatformInput = React.memo(function ConnectedPlatformInput(
     setLastUnsentText(unsentText)
     if (unsentText !== undefined) {
       lastTextRef.current = unsentText
-      setTextInput(unsentText)
     }
   }
+  // needs to be an effect since setTextInput needs a mounted ref
+  React.useEffect(() => {
```

Comment:
this is a fix for the root v630 branch changes which tried to remove most useEffects. In this case we do still need a useEffect as we need our refs to be set after rendering so we can update the component, else the `ref.current` is still null here